src/install.rst

```diff
@@ -6,197 +6,314 @@ Installing Nextstrain
+         b. Open the download file and follow through installation prompts.
+         c. Once the Visual Studio Installer opens, click **Modify** next to **Launch**, **More**, etc.
+         d. Select **Desktop development with C++**.
+         e. Go to the Individual components tab.
+         f. Search for and select **Windows 10 SDK** and the entry that ends with **C++ x64/x86 build tools (Latest)**.
+         g. Click **Modify** and follow through prompts.
 
-    Whenever you open a new terminal window to work on a Nextstrain analysis, remember to activate the Nextstrain Conda environment with ``conda activate nextstrain``.
-    Next, try :doc:`tutorials/quickstart`.
+      3. Open an Anaconda PowerShell Prompt, which can be found in the Start menu.
+      4. Install mamba on the ``base`` conda environment:
 
-Upgrade an existing installation
-================================
+         .. code-block:: bash
 
-.. tabs::
+            conda install -n base -c conda-forge mamba --yes
+            conda activate base
 
-   .. group-tab:: Conda
+      5. Create a conda environment named ``nextstrain`` and install the Nextstrain CLI:
 
-      Update the base Conda environment.
+         .. code-block:: none
 
-      .. code-block:: bash
+            mamba create -n nextstrain -c bioconda nextstrain-cli --yes
+            conda activate nextstrain
 
-         conda update -n base conda
+      6. Install the remaining Nextstrain components by `installing Docker Desktop with WSL 2 backend <https://docs.docker.com/desktop/windows/wsl/>`_.
 
-      Update the Nextstrain environment.
+         .. note::
 
-      .. code-block:: bash
+            You may have to restart your machine when configuring WSL (Windows Subsystem for Linux).
+            If so, remember to open a new Anaconda PowerShell Prompt and run ``conda activate nextstrain`` before the next step.
 
-         conda activate nextstrain
-         conda update --all
+      7. Confirm that the installation worked.
 
-   .. group-tab:: Docker
+         .. code-block:: none
 
-      Update the base Conda environment.
+            nextstrain check-setup --set-default
 
-      .. code-block:: bash
+         The final output from the last command should look like this:
 
-         conda update -n base conda
+         .. code-block:: none
 
-      Update the Nextstrain CLI package.
+            Setting default environment to docker.
 
-      .. code-block:: bash
+   .. group-tab:: WSL on Windows
 
-         conda activate nextstrain
-         conda update nextstrain-cli
+      .. note::
 
-      Download the latest image with the Nextstrain CLI.
+         If you are an experienced user, you can replace ``conda`` with ``pip`` but :doc:`note the extra installation steps for augur <augur:installation/installation>` and :doc:`install auspice via npm <auspice:introduction/install>`.
 
-      .. code-block:: bash
+      1. `Install WSL 2 <https://docs.microsoft.com/en-us/windows/wsl/install>`_.
```

Comment:
Might be possible to simplify this step by telling users to download/install something from Microsoft Store. Will need to revisit.

Reply:
Moved to #82